threeML/bayesian/bayesian_analysis.py

```diff
@@ -584,6 +608,181 @@ def corner_plot(self, **kwargs):
 
             raise RuntimeError("You have to run the sampler first, using the sample() method")
 
+    def corner_plot_cc(self, sigmas=[0, 1, 2, 3], cloud=False, shade=True, shade_alpha=1., parameters=None,
```

Comment:
You should use the reStructuredText style for docstrings, as the rest of 3ML. Can you fix this and the other docstrings and request a merge again?